bucket/xenu.json

```diff
@@ -0,0 +1,20 @@
+{
+    "homepage":  "http://home.snafu.de/tilman/xenulink.html",
+    "description": "Xenu's Link Sleuth checks Web sites for broken links",
+    "version":  "1.3.8",
+    "license":  "Unknown",
+    "url": "http://home.snafu.de/tilman/XENU.ZIP#/dl.7z",
+    "hash": "sha256:319108b72709e32c250bb49bd02739f26d37b8a9d5a9bb3bbfa488d7354e197f",
+    "installer": {
+        "script": [
+            "Expand-7zipArchive \"$dir\\Setup.exe\" \"$dir\"",
+            "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Setup.exe\", \"$dir\\`uninst.exe\" -Force -Recurse"
+        ]
+    },
+    "shortcuts": [
+        [
+            "Xenu.exe",
+            "Xenu"
+        ]
+    ]
+}
```

Comment:
Would need autoupdate or checkver

Reply:
I don't think *checkver* is needed in this case.
`(current version: 1.3.8 from September 4th, 2010)`
http://home.snafu.de/tilman/xenulink.html